### PR TITLE
Adds Judaism Mechanics

### PR DESCRIPTION
**Pull request by HarrierBernstein76**
## About The Pull Request
Possum approved this :)
<details>
It adds much needed representation on Sojourn.
<summary>
	
</summary>
<hr>

	
<hr>
</details>

## Changelog
:cl:

/:cl:

### CODE REVIEW

File: code/modules/judaism/rabbi.dm [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+/datum/job/rabbi
+	title = "Rabbi"
+	flag = RABBI
+	department_flag = CIVILIAN
+	faction = "Station"
+	total_positions = 1
+	spawn_positions = 1
+	supervisors = "G-d"
+	selection_color = "#dddddd"
+	access = list(ACCESS_SYNAGOGUE, ACCESS_MAINT_TUNNELS)
```
Comment (Trilbyspaceclone) [inline code/modules/judaism/rabbi.dm:55]: ACCESS_SYNAGOGUE isnt an access

Comment (HarrierBernstein76) [reply code/modules/judaism/rabbi.dm:55]: Oh, right we'll have to rename the church to the synagogue

**Comment by cdb-is-not-good** [comment]
Absolutely not.

**Comment by HarrierBernstein76** [comment]
I realized what I messed up, I forgot to define the foreskin organ slot.  I'll keep working on this.

**Pull request closed without merging**